week1/task2/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿// See https://aka.ms/new-console-template for more information
+string input = "Intellectualization";
+char[] vowels = new char[] { 'a', 'e', 'i', 'o', 'u' };
+int GetVowelCount(string text)
+{
+    return text.ToLower().Count(x => vowels.Contains(x));
+}
+int vowelCount = GetVowelCount(input); //TODO: Implement GetVowelCount 
+Console.WriteLine($"Number of vowels: {vowelCount}");
```

Comment:
I was expecting people to try to solve this with a foreach, If, switch case, but you found a better solution with LINQ. 
Well done.